webapp/src/main/resources/template.soy

```diff
@@ -0,0 +1,16 @@
+{namespace validator.templates}
+
+/**
+ * Renders a basic explanation of the validation notice.
+ * @param numNotices
+ * @param notices
+ */
+{template .unknownColumnNotice}
+  <p class="warning">Warning - Unknown Column(s) found!</p>
+  <p><b>{$numNotices}</b> unknown column(s) found in:</p>
+  {foreach $notice in $notices}
+    <p>Filename: <i>{$notice.filename}</i></p>
+    <p>Column name: <i>{$notice.fieldName}</i> at position <i>{$notice.index}</i></p>
```

Comment:
These fields are not guaranteed to appear for every notice and some other fields are included for different notices. Also, for example, some notice may involve more than 1 file. But it looks fine as a beginning template example.

Reply:
This template is just for the unknown column notice. So these fields corresponds to that.

Reply:
Sure.